packages/orbit-components/src/Textarea/__tests__/index.test.tsx

```diff
@@ -49,8 +49,8 @@ describe("Textarea", () => {
     expect(textarea).toHaveAttribute("name", name);
     expect(textarea).toHaveAttribute("data-sample", "Sample");
     expect(textarea).not.toBeInvalid();
-    expect(textarea.parentElement).toHaveStyle({ marginBottom: "12px" });
     expect(textarea).toHaveStyle({ padding: "12px" });
+    expect(document.querySelector(".orbit-textarea")).toHaveStyle({ marginBottom: "12px" });
```

Comment:
❓ Is the only reason for this change to have a "proper" targeting of the element?

Reply:
Not sure what you mean by `"proper" targeting` but before it was targeting the `parentElement` of the textarea but with my change the textarea is one level deeper so it makes more sense to target it like this 🤔

Reply:
yes, I get it. I did some local changes, so it worked with `parentElement` and that was a reason why I asked.